pages/_app.js

```diff
@@ -1,10 +1,23 @@
 import GlobalStyle from "../styles";
+import useLocalStorageState from "use-local-storage-state";
+import data from "@/db/data";
 
 export default function App({ Component, pageProps }) {
+  const [storedRecipes, setRecipe] = useLocalStorageState("recipes", {
+    defaultValue: [...data],
+  });
```

Comment:
- ` storedRecipes` suggests that it's an array of recipes, which is good. However, `setRecipe` (singular) might imply that it's a function for setting a` single recipe, not updating the entire array of recipes`
- **Much more important**:  Use the `[noun, setNoun]`  pattern convention for React correctly.  🥹 _Note: Please do not forget to adjust the properties when renaming._

```suggestion
  const [recipes, setRecipes] = useLocalStorageState("recipes", {
    defaultValue: [...data],
  });
```